service/health/searcher/service.go

```diff
@@ -29,6 +38,14 @@ func New(config Config) (*Service, error) {
 	if config.Provider == "" {
 		return nil, microerror.Maskf(invalidConfigError, "provider must not be empty")
 	}
+	providers := []string{
```

Comment:
This should be a `const` - or maybe in a `key` function?

Reply:
Moved to key.